assets/src/edit-story/components/canvas/framesLayer.js

```diff
@@ -98,15 +101,28 @@ function FramesLayer() {
     state: { draggingResource, dropTargets },
     actions: { isDropSource },
   } = useDropTargets();
+  const { setScrollOffset } = useLayout(({ actions: { setScrollOffset } }) => ({
+    setScrollOffset,
+  }));
 
   const ref = useRef(null);
   useCanvasKeys(ref);
 
+  const onScroll = useCallback(
+    (evt) =>
+      setScrollOffset({
+        left: evt.target.scrollLeft,
+        top: evt.target.scrollTop,
+      }),
+    [setScrollOffset]
+  );
+
   return (
     <Layer
       ref={ref}
       data-testid="FramesLayer"
-      pointerEvents="none"
```

Comment:
Hm, I wonder if it was `none` before for a specific purpose.

Reply:
Ditto, but couldn't find any problem with removing it. And it's needed to allow scroll events. Drag-to-select and everything else still seems to work. It's hopefully something that was necessary once, but not anymore.

Reply:
Has this PR merged clicking through the empty area of a shape (#2038)? Could you merge `main` to ensure we're testing up to date features?

Reply:
I just merged `main` in. Let me know if you experience regressions.

Reply:
Hm, I’m not able to access image edit mode currently, working ok for you?

Reply:
![edit](https://user-images.githubusercontent.com/3294597/106620815-3aac9c80-6572-11eb-99b7-35029830cd6c.gif)

Reply:
^ Ignore this, seems to happen on `main`, too!

Reply:
So, this original line of code did have an impact on the video control element, which [needed a manual `pointer-events: none`](https://github.com/google/web-stories-wp/pull/6112/commits/2e80ecebca8ebb7f4da657920cee09e2903c2856), but other than that, everything else seems to work.